roles/StackStorm.ewc/tasks/rbac.yml

```diff
@@ -54,6 +54,15 @@
     value: True
     backup: yes
   notify:
-    - restart st2
-    - reload ewc_rbac
+    - restart st2api
+
+- name: Configure RBAC backend in st2 configuration
```

Comment:
This should be removed as it’s part of the deb/rpm packaging scriplets.
Duplicating steps doesn’t help but instead makes harder to find issues in package installation results.

Reply:
I think it's still better / safer to leave it here.

It doesn't hurt and it won't trigger a notification anyway if the option is already present.